command/info_sub/variables.sh

```diff
@@ -56,147 +55,130 @@ See also:
 
 ' 1
 
-if ! typeset -F getopts_long >/dev/null 2>&1 ; then
+if ! typeset -F getopts_long >/dev/null 2>&1; then
+    # shellcheck source=./../../getopts_long.sh
     . "${_Dbg_libdir}/getopts_long.sh"
 fi
 
-# Should declare typset_flags before calling.
-# That it the implit returned value
-function _Dbg_info_variables_parse_options {
-
-    typeset -i _Dbg_rc=0
+function _Dbg_do_info_variables() {
+    declare _Dbg_typeset_flags=""
+    # 0: print variables with flags
+    # 1: filter by flags in $_Dbg_typeset_flags and don't print flags
+    declare -i _Dbg_typeset_filtered=1
+    _Dbg_info_variables_parse_options "$@"
+    (($? != 0)) && return
+
+    # Caveats:
+    #   Bash >= 5.2: 'declare -p' properly escapes special characters within $'', e.g. $'\n\t'
```

Comment:
A suggestion is to remove the information about Bash >= 5.2 here because it is not relevant. Same with "To work with all of Bash "5.x" below. 

These comments exist in the 5.2 branch, so there is no loss as far as I can tell.

Your thoughts?

Reply:
I'll rephrase this for bash 4.4

Reply:
Done, finally